frontend/src/components/LayoutItem.module.css

```diff
@@ -0,0 +1,37 @@
+.item {
+  cursor: pointer;
+  background-color: #c5ced2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0px 20px 20px 0px;
+  flex-grow: 1;
+  padding: 0 15px;
+  text-decoration: none;
+  color: #000;
+}
+
+.item_top {
+  border-top-left-radius: 20px;
+}
```

Comment:
This border should be handled by parent container. It will cut overflow of this element too.